projects/aca-content/assets/i18n/ar.json

```diff
@@ -486,7 +486,7 @@
   },
   "INFO_PANEL": {
     "TABS": {
-    
+
```

Comment:
empty line probably not needed

Reply:
removed whole TABS block so no empty line anymore